lib/helpers/remote_control_helper.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+require 'net/ssh'
+require 'net/sftp'
+require 'stringio'
+require 'logger'
+
+# @return [Logger]
+def logger
+  @logger = Logger.new($stdout)
+end
+
+# @param [Object] host
+# @param [Object] user
+# @param [Proc] block
+#
+# @return [Array, Net::SSH::Authentication]
+def ssh(host, user, &block)
+  @ssh = Net::SSH.start(host, user, &block)
+end
+
+# Describer
+class RemoteControlHelper
+
+  attr_reader :docserver_version, :spec, :host
+
+  # @param [Object] host
+  # @param [Object] docserver_version
+  # @param [Object] spec_name
+  # 
+  # @return [Object]
+  def initialize(host, docserver_version, spec_name)
+    @host = host
+    @docserver_version = docserver_version
+    @spec_name = spec_name
+  end
+
+  private
+
+  # @param [Object] session
+  # @param [Object] script
+  # @param [String] shell
+  #
+  # @return [Object]
+  def execute_in_shell!(session, script, shell = 'bash')
+    channel = session.open_channel do |ch|
+      ch.exec("#{shell} -l") do |ch2, success|
+        raise "could not execute command" unless success
+        # Set the terminal type
+        ch2.send_data 'export TERM=vt100n'
+        # Output each command as if they were entered on the command line
+        [script].flatten.each do |command|
+          ch2.send_data "#{command}n"
+        end
+        # Remember to exit or we'll hang!
+        ch2.send_data 'exitn'
+        # Configure to listen to ch2 data so you can grab stdout
+      end
+    end
+    channel.wait
+  end
+end
+
+# @param [Object] session
+# @param [Object] path
+#
+# @return [String]
+def download!(session, path)
```

Comment:
There is a mess from Global methods and `RemoteControlHelper` methods in one file, I think all global methods should be extracted to some helper

Maybe SftpHelper wth `download`, `upload` and other methods